aggregator/api.py

```diff
@@ -1,33 +1,32 @@
-from flask import Flask
-from flask import json
 import time
 import requests
 from datetime import datetime
+import json
+import psycopg2
+import os
 
-app = Flask(__name__)
+DATABASE_URL = os.getenv("DATABASE_URL")
```

Comment:
It's a bit difficult to figure out how to start and run the aggregator. Could be useful to make a script for setting up the .env and to place the actual path instead of an environment variable and to clarify in a comment that we can use dev database in neon

Reply:
I don't quite understand the comment. There is a "run.sh" script that sources the `.env` file and runs the script. Of course the `.env` can't be pushed to github as it contains database credentials and should be shared with the team through other means.